Framework/Kernel/inc/MantidKernel/SetValueWhenProperty.h

```diff
@@ -0,0 +1,56 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+#pragma once
+
+#include "MantidKernel/IPropertySettings.h"
+#include "MantidKernel/System.h"
```

Comment:
Can we replace this with `#include "MantidKernel/DllExport.h"`?

Reply:
#include "MantidKernel/DllConfig.h"